examples/keras_io/nlp/text_classification_with_transformers.py

```diff
@@ -0,0 +1,115 @@
+"""
+Title: Text Classification with Transformers
+Author: [Yash Srivastava](https://twitter.com/Yaaaaaashhh)
```

Comment:
Please keep the original author, you can check the 4th point in Stage 1: https://github.com/keras-team/keras-core/issues/222

Reply:
I think I misread that. I'll add the original author's name.